h/security/__init__.py

```diff
@@ -10,11 +10,7 @@
 from h.security.identity import Identity  # noqa:F401
 from h.security.permissions import Permission  # noqa:F401
 from h.security.permits import identity_permits
-from h.security.policy.top_level import BearerTokenPolicy, TopLevelPolicy
-
-# We export this for the websocket to use as it's main policy
-__all__ = ("BearerTokenPolicy",)
-
+from h.security.policy import StreamerPolicy, TopLevelPolicy
```

Comment:
`security/__init__.py` never should have been importing `BearerTokenPolicy` from `policy/top_level.py`, weird. Fixed it to use `policy/__init__.py` properly instead, so that the internal structure of the `policy` subdir is private to it